package.json

```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "buffer-reverse": "^1.0.1",
     "buffer-xor": "^2.0.2",
-    "sodium-native": "^2.4.2"
+    "sodium-native": "^2.4.2",
+    "tape": "^4.10.1"
```

Comment:
You want to move this one to `devDependencies`.
You can install directly to dev dependencies with `npm install --save-dev tape`. Dev dependencies are not installed for dependants, eg. someone depending on minisign will not need tape to run it.